mono/metadata/unity-liveness.c

```diff
@@ -106,79 +156,54 @@ const int kArrayElementsPerChunk = 256;
 const int kMaxTraverseRecursionDepth = 128;
 
 /* Liveness calculation */
-MONO_API LivenessState* mono_unity_liveness_allocate_struct (MonoClass* filter, guint max_count, register_object_callback callback, void* callback_userdata, WorldStateChanged onWorldStartCallback, WorldStateChanged onWorldStopCallback);
-MONO_API void           mono_unity_liveness_stop_gc_world (LivenessState* state);
-MONO_API void           mono_unity_liveness_finalize (LivenessState* state);
-MONO_API void           mono_unity_liveness_start_gc_world (LivenessState* state);
-MONO_API void           mono_unity_liveness_free_struct (LivenessState* state);
-
-MONO_API LivenessState* mono_unity_liveness_calculation_begin (MonoClass* filter, guint max_count, register_object_callback callback, void* callback_userdata, WorldStateChanged onStartWorldCallback, WorldStateChanged onStopWorldCallback);
-MONO_API void           mono_unity_liveness_calculation_end (LivenessState* state);
-
-MONO_API void           mono_unity_liveness_calculation_from_root (MonoObject* root, LivenessState* state);
-MONO_API void           mono_unity_liveness_calculation_from_statics (LivenessState* state);
-
-#define MARK_OBJ(obj) \
-	do { \
-		(obj)->vtable = (MonoVTable*)(((gsize)(obj)->vtable) | (gsize)1); \
+MONO_API LivenessState * mono_unity_liveness_allocate_struct(MonoClass *filter, guint max_count, register_object_callback callback, void *callback_userdata, ReallocateArray reallocateArray);
+MONO_API void mono_unity_liveness_stop_gc_world();
+MONO_API void mono_unity_liveness_finalize(LivenessState *state);
+MONO_API void mono_unity_liveness_start_gc_world();
+MONO_API void mono_unity_liveness_free_struct(LivenessState *state);
+
+MONO_API void mono_unity_liveness_calculation_from_root(MonoObject *root, LivenessState *state);
+MONO_API void mono_unity_liveness_calculation_from_statics(LivenessState *state);
+
+#define MARK_OBJ(obj)                                                       \
+	do {                                                                    \
+		(obj)->vtable = (MonoVTable *)(((gsize) (obj)->vtable) | (gsize)1); \
 	} while (0)
 
-#define CLEAR_OBJ(obj) \
-	do { \
-		(obj)->vtable = (MonoVTable*)(((gsize)(obj)->vtable) & ~(gsize)1); \
+#define CLEAR_OBJ(obj)                                                       \
```

Comment:
Any reason for the whitespace changes?

Reply:
No. I suppose VS did this behind my back